modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_move_boxes.js

```diff
@@ -283,11 +289,23 @@ AUI.add(
 						var btnDisabledRight = true;
 
 						if (target.get('length') > 0) {
-							if (target == instance._rightBox) {
-								btnDisabledLeft = false;
+							if (target === instance._rightBox) {
```

Comment:
I think we cannot add the logic here because this method is executed after moving the items right? But potentially we could move several items at once. Imagine that you limit the right box to only hold 2 items, but if you have already one, and select two from the left it will allow moving them, which is not the behaviour that we want.

That example is illustrated here:
![example](https://user-images.githubusercontent.com/11654230/87415021-db367180-c5cc-11ea-9782-1b1195179ec4.gif)

In this case as soon as the user select two items on the left the move button should be disabled

Reply:
@victorg1991, updated this, can you check to see if this address the issue? Can you see anything else we might've missed?